Content.Server/Storage/Components/EntityStorageECSComponent.cs

```diff
@@ -0,0 +1,129 @@
+using Content.Shared.Acts;
+using Content.Shared.Physics;
+using Content.Shared.Sound;
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+using System;
+using System.Linq;
+
+namespace Content.Server.Storage.Components
+{
+    [RegisterComponent]
+    public class EntityStorageECSComponent : Component, IExAct
```

Comment:
why does it have ECS on its name? temporary?

Reply:
yes, temporary